build.sbt

```diff
@@ -78,6 +78,10 @@ lazy val ops = project
     name := "ammonite-ops"
   )
 
+lazy val prompt = shellPrompt in ThisBuild := { state =>
```

Comment:
What prompt are you talking about? Why do we care about the submodule's prompt? Why is this prompt better?